docs/development/affiliated-packages.rst

```diff
@@ -269,7 +269,7 @@ files manually`_ section since this explains what many of the files do.
       or just delete it.
    #. Exit out of your text editor.
 
-#. Update the main package docstring in ``<packagename>/__init__.py``.
+#. Update the main package docstring in ``<packagename>/packagename/__init__.py``.
```

Comment:
I still see it at https://github.com/astropy/package-template/tree/master/packagename